eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -114,4 +114,21 @@ public QueryResponse getVariantsByRegion(@PathVariable("regionId") String region
     public QueryResponse getVariantsByRegion() {
         return setQueryResponse("");
     }
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
+    @ResponseBody
+//    @ApiOperation(httpMethod = "GET", value = "Retrieves all the variants from region", response = QueryResponse.class)
+    public QueryResponse getChromosomes(@RequestParam(name = "species") String species,
+                                        HttpServletResponse response)
```

Comment:
In the vcf-dumper WS, study(ies) is a mandatory param (and files for the CLI). If the query speed is not affected by adding those filters, it would improve the vcf-dumper performance (for studies that don't have variants in each chromosome). If the speed of this query is affected, maybe is better not executing those filters, so we get the list of chrosomes and start exporting faster (this would be good for the WS, when a long waiting time since we call the WS until we get some data can confuse the user).